### PR TITLE
4.10 api changes and fixed issues 

### PR DESCRIPTION
**Pull request by karuturi**
copied from list generated by @swill

### CODE REVIEW

**Comment by karuturi** [comment]
@swill @pdion891 please review

**Comment by swill** [comment]
LGTM @karuturi 👍

File: source/fixed_issues.rst [hunk 1]
```diff
@@ -28,1097 +29,1490 @@ Issues Fixed in |release|
...
-| 4.9                | `#1458`_ | CLOUDSTACK-9331_   | Bug           | Major    | Automation: Prepare and add the baremetal cfg to           |
-|                    |          |                    |               |          | marvin/config folder &marvin frame work changes to support |
-|                    |          |                    |               |          | baremetal advanced testcase                                |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1452`_ | CLOUDSTACK-9322_   | Task          | Major    | Support for Internal LB functionality with Nuage VSP SDN   |
-|                    |          |                    |               |          | Plugin including Marvin test coverage                      |
-+--------------------+----------+--------------------+---------------+----------+------------------------------------------------------------+
-| 4.9                | `#1271`_ | CLOUDSTACK-9164_   | Bug           | Major    | Consoleproxy does not preven
```
Comment (DaanHoogland) [inline source/fixed_issues.rst]: @karuturi this should read "| 4.10                    | `#2071`_ | CLOUDSTACK-9815_   | Improvement     | Major    | Changed location of certificate service |"
the full feature isn't implemented yet

Comment (karuturi) [inline source/fixed_issues.rst]: @DaanHoogland updated

**Comment by karuturi** [comment]
merging this now

**Pull request merged**